hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/distributions/dstable/DistributionSetTableHeader.java

```diff
@@ -50,49 +55,51 @@ void onEvent(final DistributionsUIEvent event) {
 
     @Override
     protected String onLoadSearchBoxValue() {
-        return manageDistUIstate.getManageDistFilters().getSearchText().orElse(null);
+        return getManageDistUIstate().getManageDistFilters().getSearchText().orElse(null);
     }
 
     @Override
     protected void showFilterButtonsLayout() {
-        manageDistUIstate.setDistTypeFilterClosed(false);
+        getManageDistUIstate().setDistTypeFilterClosed(false);
```

Comment:
I changed the scope from protected to private, and provided a getter for accessing. As far as I know we agreed to do this like that. The getter is in AbstractTableHeader.java